academy/ibc-dev/ibcapp_steps.md

```diff
@@ -0,0 +1,425 @@
+# Make a module IBC-enabled
+
+In this section, you'll build a conceptual SDK blockchain with one module. The first time, a regular module and the second time an IBC module. This will introduce us to what makes a module IBC-enabled.
+
+## Scaffold a Leaderboard chain
+
+By now, you are familiar with scaffolding a chain with Ignite CLI (if not, check out the [Create Your Own Chain](insert_link) section).
+
+Let's scaffold a `leaderboard` chain:
+
+```bash
+ignite scaffold chain github.com/cosmonaut/leaderboard
```

Comment:
In the rest of the content, we prefix shell commands with `$`. The "Copy" button removes it already.